### DISCUSSION TITLE
Do 'createSelector' can access to the previous value of the state inside the function ?

### DISCUSSION THREAD

**Discussion by iamAravindks** [ngrx/platform#4351] [Q&A]
### Which @ngrx/* package(s) are relevant/related to the feature request?

data, store

### Information

I intend to capture the previous state within the createSelector or similar functionality. How can I achieve this? I couldn't find any relevant information. Would anyone be able to help me with this?

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by brandonroberts** [comment]
Selectors themselves do not have access to previous values. What's the use case?
